### PR TITLE
Remove git repo deps

### PR DESCRIPTION
**Pull request by TroyKomodo**
## Proposed changes

Moves all git repo deps to fixed crates.io crates.

## Types of changes

What types of changes does your code introduce to Scuffle?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](https://github.com/ScuffleTV/scuffle/blob/main/CONTRIBUTING.md) doc
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, you may want to start the discussion by explaining why you chose the solution you did, what alternatives you considered, etc.

### CODE REVIEW

**Review by SputNikPlop — APPROVED** [review]

File: common/src/logging.rs [hunk 1]
```diff
@@ -29,26 +30,84 @@ pub enum LoggingError {
...
+
+				filter.finish().try_init()?;
+
+				Ok(Box::new(move |level: &str| {
+					let level = EnvFilter::from_str(level)?;
+					handle.reload(level)?;
+					Ok(())
+				}) as ReloadHandle)
+			}
 		}
```
Comment (lennartkloock) [inline common/src/logging.rs:107]: Why is there so much code duplication here? Can't this be solved in a more elegant way?

File: platform/image_processor/src/processor/job/resize.rs [hunk 2]
```diff
@@ -38,30 +28,17 @@ impl std::fmt::Debug for ImageResizer {
...
-
 impl ImageResizer {
 	pub fn new(target: ImageResizerTarget) -> Self {
 		Self {
+			resizer: fr::Resizer::new(match target.algorithm {
+				ResizeAlgorithm::Nearest => fr::ResizeAlg::Nearest,
+				ResizeAlgorithm::Linear => fr::ResizeAlg::Convolution(fr::FilterType::Bilinear),
+				ResizeAlgorithm::Cubic => fr::ResizeAlg::Convolution(fr::FilterType::CatmullRom),
+				ResizeAlgorithm::Area => fr::ResizeAlg::Convolution(fr::FilterType::Box),
+				ResizeAlgorithm::Lanczos4 => fr::ResizeAlg::Convolution(fr::FilterType::Lanczos3),
```
Comment (lennartkloock) [inline platform/image_processor/src/processor/job/resize.rs:39]: Why is Lanczos4 actually Lanczos3?

File: common/src/logging.rs (hunk 1 above)
Comment (TroyKomodo) [reply common/src/logging.rs:107]: Unfortunately no, tracing has layers, by changing `.json()` or `.compact()` or `.pretty()` it changes a root layer meaning the types are different on the match branches.

Comment (TroyKomodo) [reply common/src/logging.rs:107]: Not without a macro.

File: platform/image_processor/src/processor/job/resize.rs (hunk 2 above)
Comment (TroyKomodo) [reply platform/image_processor/src/processor/job/resize.rs:38]: We can rename the proto, the pure rust impl does not have Lanczos4 only Lanczos3

**Review by lennartkloock — APPROVED** [review]

**Review by treuks — APPROVED** [review]

**Pull request merged**